torch/_dynamo/config.py

```diff
@@ -98,7 +98,7 @@
 # This is a good way to get your model to work one way or another, but you may
 # lose optimization opportunities this way.  Devs, if your benchmark model is failing
 # this way, you should figure out why instead of suppressing it.
-suppress_errors = bool(os.environ.get("TORCHDYNAMO_SUPPRESS_ERRORS", False))
+suppress_errors = bool(os.environ.get("TORCHDYNAMO_SUPPRESS_ERRORS", True))
```

Comment:
```suggestion
suppress_errors = os.environ.get("TORCHDYNAMO_SUPPRESS_ERRORS", "1")=="1"
```

Prior code would confusingly mean `TORCHDYNAMO_SUPPRESS_ERRORS=0` does nothing.